Source/Core/Core/BootManager.cpp

```diff
@@ -411,14 +394,41 @@ void Stop()
   RestoreConfig();
 }
 
+// SYSCONF can be modified during emulation by the user and internally, which makes it
+// a bad idea to just always overwrite it with the settings from the base layer.
+//
+// Conversely, we also shouldn't just discard the base layer, as it may cause
```

Comment:
instead of "we also shouldn't just discard the base layer," perhaps "we also shouldn't just ignore any changes to SYSCONF,"